matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/call/model/MxCallImpl.kt

```diff
@@ -203,8 +203,10 @@ internal class MxCallImpl(
 
     override fun selectAnswer() {
         Timber.tag(loggerTag.value).v("select answer $callId")
-        if (isOutgoing) return
-        state = CallState.Answering
+        if (!isOutgoing) return
+        
+        // This is an outgoing call, select the remote client that answered.
+        // state is still DIALING.
```

Comment:
Do we want to actively reinforce that assumption at line 206?

Reply:
You think just remove the line about `state`?

Reply:
Your comment says that `state is still DIALING`
So I am wondering if it was worth  adding
```kotlin
if (state != CallState.Dialing) return
```

Reply:
(Or log a warning)

Reply:
Thanks